### PR TITLE
Psp 2378 bc assessment tab(s)

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: source/frontend/src/features/mapSideBar/tabs/bcAssessment/BcAssessmentTabView.test.tsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+import { IFormLease } from 'interfaces';
+import { render, RenderOptions } from 'utils/test-utils';
+
+import BcAssessmentTabView, { IBcAssessmentTabViewProps } from './BcAssessmentTabView';
+
+const history = createMemoryHistory();
+
+describe('BcAssessmentTabView component', () => {
+  const setup = (
+    renderOptions: RenderOptions & IBcAssessmentTabViewProps & { lease?: IFormLease } = {
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/tabs/bcAssessment/BcAssessmentTabView.test.tsx:12]: do we need **lease* here in this type? seems like copy/pasta

File: source/frontend/src/hooks/useBcAssessmentLayer.ts [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+      }
+      if (process.env.NODE_ENV === 'development') {
+        return {
+          data: mockBcAssessmentSummary,
+          status: 200,
+          statusText: 'Success',
+          headers: {},
+          config: {},
+        };
+      }
```
Comment (asanchezr) [inline source/frontend/src/hooks/useBcAssessmentLayer.ts:95]: this is certainly a new pattern for local development - I gather we didn't want to setup a local WFS layer in geoserver for this?

**Review by asanchezr — APPROVED** [review]

File: source/frontend/src/hooks/useBcAssessmentLayer.ts (hunk 2 above)
Comment (devinleighsmith) [reply source/frontend/src/hooks/useBcAssessmentLayer.ts:95]: we don't have the underlying data in this case.

Comment (devinleighsmith) [reply source/frontend/src/hooks/useBcAssessmentLayer.ts:95]: Logged https://jira.th.gov.bc.ca/browse/PSP-4852 as a follow up, should be possible to request access to the underlying layers.

**Pull request merged**